Doc/library/sqlite3.rst

```diff
@@ -406,14 +406,21 @@ Connection Objects
 
    .. attribute:: isolation_level
 
-      Get or set the current default isolation level. :const:`None` for autocommit mode or
-      one of "DEFERRED", "IMMEDIATE" or "EXCLUSIVE". See section
-      :ref:`sqlite3-controlling-transactions` for a more detailed explanation.
+      Get or set the isolation level.
+      Set to :const:`None` to disable implicit transaction handling,
+      or to one of the ``"DEFERRED"``, ``"IMMEDIATE"``, or ``"EXCLUSIVE"``
+      transaction handling modes.
```

Comment:
There is one thing that still disturbs me with this part: it is not technically correct to say that DEFERRED, IMMEDIATE, and EXCLUSIVE are transaction handling modes; this may be percieved as if sqlite3 has four transaction handling modes (one for each allowed value). However, it has only two:

1. *isolation_level* is `None`: no implicit transaction handling is performed (except for the implicit commit in `executescript`)
2. *isolation_level* is a string: implicit transaction handling is performed (implicit BEGIN before INSERT, DELETE, etc.)

Perhaps it is better to describe these as:

1. *isolation_level* is `None`: transactions are never implicitly opened
2. *isolation_level* is `str`: transactions are implicitly opened before INSERT, etc.

The three allowed string values relates to low level SQLite [transaction behaviour](https://www.sqlite.org/lang_transaction.html), but the *sqlite3* implicit transaction mode is the same, no matter the *SQLite* transaction *behaviour*.

I'm not sure how to best communicate these things, but we cannot land this PR yet :(

Reply:
I tried to address all of this in 1473e948d54b18755dcef8daf421e11f48d7ebf7.